airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -328,6 +328,7 @@ def __init__(
         is_delete_operator_pod: None | bool = None,
         termination_message_policy: str = "File",
         active_deadline_seconds: int | None = None,
+        progress_callback: Callable[[str], None] | None = None,
```

Comment:
Please add this parameter to the class doc string description 
`:param progress_callback: ...`